src/main/java/com/minecolonies/client/gui/WindowHutFarmer.java

```diff
@@ -1,198 +1,187 @@
 package com.minecolonies.client.gui;
 
+import com.blockout.Pane;
 import com.blockout.controls.Button;
 import com.blockout.controls.ButtonVanilla;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.blockout.controls.ButtonVanilla'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUselessImportCheck)